Storage/Storage/Model/MIGRATIONS.md

```diff
@@ -2,6 +2,11 @@
 
 This file documents changes in the WCiOS Storage data model. Please explain any changes to the data model as well as any custom migrations.
 
+## Model 30 (Release 5.0.0.0)
+- @ecarrion 2020-09-02
+- Deletes `OrderStats` entity
+- Deletes `OrderStatsItem` entity
```

Comment:
super nit: the previous migration notes seem to be following just the simple present tense, maybe can update "Deletes" to "Delete"?
```suggestion
- Delete `OrderStats` entity
- Delete `OrderStatsItem` entity
```